src/components/bottom-bar-item/bottom-bar-item.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+
+import { BottomBarItem } from './bottom-bar-item'
+import { FlexContainer } from '../layout'
+import { Icon } from '../icon'
+import {
+  ElAnchorBottomBarItemContainer,
+  ElBottomBarItemIcon,
+  ElBottomBarItemLabel,
+  ElButtonBottomBarItemContainer,
+} from './styles'
+
+const meta = {
+  title: 'Components/Bottom Bar Item',
+  component: BottomBarItem,
+  parameters: {
+    layout: 'padded',
+  },
+} satisfies Meta<typeof BottomBarItem>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    isActive: false,
+  },
+  render: (args) => {
+    return (
```

Comment:
**question:** Can you help me understand your reasoning behind this component API? Specifically, why do you have `Icon` and `Label` as separate components to the main `BottomBarItem`?

Reply:
> Can you help me understand your reasoning behind this component API?

at first, I thought the `BottomBarItem` would also handle the floating menu, as described [here](https://www.figma.com/design/XJ6qcAV8gHscsUodqJMNEF/Reapit-Elements-production-ready-components?node-id=34-6931&node-type=instance&m=dev). However, I was wrong. This functionality should be handled neither by the `BottomBar` nor any upper parent component